components/TextControls/BUILD

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017-present The Material Components for iOS Authors. All Rights Reserved.
+# Copyright 2020-present The Material Components for iOS Authors. All Rights Reserved.
```

Comment:
did we land this file in 2019?

Reply:
Oh yes, we did. Should I change it to 2019?

Reply:
yep that would be more accurate. (since your changing it :)

Reply:
Done